packages/xo-server/src/xapi/index.js

```diff
@@ -894,6 +913,17 @@ export default class Xapi extends XapiBase {
         this._exportVdi($cancelToken, vdi, baseVdi, VDI_FORMAT_VHD)
     })
 
+    const suspendVdi = vm.$suspend_VDI
+    if (suspendVdi !== undefined) {
+      const vdiRef = suspendVdi.$ref
+      vdis[vdiRef] = {
+        ...suspendVdi,
+        $SR$uuid: suspendVdi.$SR.uuid,
```

Comment:
I don't understand this. Who needs this property?

Reply:
I don't remember, I copied the process used to create other VDIs.

Reply:
Ok, I thought it was specific to suspend VDIs.